cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Gov/ProposeNewConstitution.hs

```diff
@@ -198,20 +199,8 @@ hprop_ledger_events_propose_new_constitution = H.integrationWorkspace "propose-n
                      (paymentKeyInfoPair wallet0:[defaultDRepKeyPair n | (_, n) <- allVotes])
   submitTx execConfig cEra voteTxFp
 
-  -- We check that constitution was succcessfully ratified
-
-  !eConstitutionAdopted
-    <- evalIO . runExceptT $ foldEpochState
-                      (File configurationFile)
-                      (File socketPath)
-                      FullValidation
-                      (EpochNo 10)
-                      ()
-                      (\epochState _ _ -> foldBlocksCheckConstitutionWasRatified constitutionHash constitutionScriptHash epochState)
-
-  void $ evalEither eConstitutionAdopted
+  _ <- waitForEpochs epochStateView (EpochInterval 1)
```

Comment:
I'm not sure if we should use `foldEpochState` to wait here for votes or simply wait for a single epoch. Single epoch wait seems easier and should be good enough imo.

Reply:
Can you add a comment explaining why we can be sure the proposal won't be removed from the ledger state after waiting an epoch?

Reply:
aded